lib/virt_autotest/virtual_network_utils.pm

```diff
@@ -91,10 +94,11 @@ sub test_network_interface {
     save_screenshot;
 
     # Restore the network interface to the default for the Xen guests
-    if (get_var('XEN') || check_var('SYSTEM_ROLE', 'xen') || check_var('HOST_HYPERVISOR', 'xen')) {
-        assert_script_run("ssh root\@$guest 'cd /etc/sysconfig/network/; cp ifcfg-eth0 ifcfg-$nic'");
+    if (!get_var("SRIOV_NETWORK_CARD_PCI_PASSSHTROUGH")) {
+        if (get_var('XEN') || check_var('SYSTEM_ROLE', 'xen') || check_var('HOST_HYPERVISOR', 'xen')) {
```

Comment:
In the longer run we probably should think about refactoring a `is_xen` routine.

Reply:
Since `is_xen_host` is introduced here in `tests/virt_autotest/virt_utils.pm` - why not use it here?

Reply:
because this test reuse functions from another test(not under the same directory) and meanwhile have to make some change to that function, I am a bit cautious not to introduce more dependencies to that test. but now I think I was too worried about it, and the directory structures may be readjusted as they are all virtualization tests.

Reply:
@Julie-CAO I agree with @grisu48 but I suggest to move those functions under `lib/` directory so we can use it everywhere!